### PR TITLE
Reduce critical PHP false positives

### LINKED ISSUE #314: probable false: combo/backdoor/php in neovim
```
/home/t/packages/x86_64/neovim-0.10/usr/bin/nvim [🚨 CRITICAL]
-------------------------------------------------------------------------
RISK  KEY                 DESCRIPTION            EVIDENCE                
-------------------------------------------------------------------------
CRIT  combo/backdoor/php  php base64 eval uname  base64_decode           
                                                 eval(::$eval            
                                                 uname()::$html_special  
-------------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by tstromberg**
Fixes #314 #287 #286 by making the matching conditions stricter.

Adds Composer sample (https://getcomposer.org/) from wolfi.

### CODE REVIEW

File: rules/evasion/script-obfuscation.yara [hunk 1]
```diff
@@ -113,9 +114,23 @@ rule php_obfuscated_concat : high {
     hash_2024_PHP_wlstncyj = "1a1c97594340ede77bc814670eaf35eaba861f1f9519038582416c704796da0a"
   strings:
     $php = /<\?[^x]/
```
Comment (egibs) [inline rules/evasion/script-obfuscation.yara:116]: Would it make sense to update this to match the other `$php` strings (`$php = "<?php"`)?

Comment (tstromberg) [reply rules/evasion/script-obfuscation.yara]: sounds good - I don't believe short tags are supported by default anywhere in 2024.

**Review by egibs — COMMENTED** [review]
> Adds Composer sample ([getcomposer.org](https://getcomposer.org/)) from wolfi.

Rules LGTM -- does this still need to be added?

**Comment by tstromberg** [comment]
oof. thanks for catching the missing samples!

**Review by egibs — APPROVED** [review]

**Pull request merged**